pincer/commands/commands.py

```diff
@@ -580,11 +618,15 @@ class ChatCommandHandler(metaclass=Singleton):
         Dictionary of managers
     register: Dict[:class:`str`, :class:`~objects.app.command.ClientCommandStructure`]
         Dictionary of ``ClientCommandStructure``
-    """
+    built_register: Dict[:class:`str`, :class:`~objects.app.command.ClientCommandStructure`]
```

Comment:
Shouldn't this be `~pincer.objects` instead?

Reply:
I think it works anyway but I'm gonna change it for consistency